docs/getting-started/cloudflare-workers.md

```diff
@@ -247,6 +247,50 @@ app.put('/upload/:key', async (c, next) => {
 })
 ```
 
+If the request handler were put into a separate file, you can pass the "_type struct_" through `Context`.
+
+```ts
+import { Context } from 'hono'
+
+type Bindings = {
+  MY_BUCKET: R2Bucket
+  USERNAME: string
+  PASSWORD: string
+}
+
+export const handler = async (c: Context<{
+  Bindings: Bindings
+}>) => {
+  return c.text(c.env.USERNAME)
+}
+```
+
+Cloudflare Wrangler also had the ability to generate types for bindings, So you can use the generated types directly in any `.ts` file.
+
+```sh
+wrangler types
+```
+
+After running the command you will get a `worker-configuration.d.ts` file which will have a typescript interface called `Env`.
+
+```ts
+// Generated by Wrangler by running `wrangler types`
+
+interface Env {
```

Comment:
The naming `Env` will be confusing because we often use the following code:

```ts
type Env = {
  Bindings: {
    KV: KVNamespace
  },
  Variables: {
    foo: string
  }
}
```

Actually, the code which is generated from [C3](https://developers.cloudflare.com/pages/get-started/c3/) does not use `Env` but uses `CloudflareBindings`:

```ts
// Generated by Wrangler by running `wrangler types --env-interface CloudflareBindings`

interface CloudflareBindings {
	MY_VARIABLE: "production_value";
}
```